src/components/search/SearchBar.tsx

```diff
@@ -176,7 +176,7 @@ const SearchBar: React.FC<SearchBarProps> = (props) => {
           onChangeHandler(newValue);
         }
       }}
-      defaultValue={props.initialValue}
+      //defaultValue={props.initialValue}
```

Comment:
I still don't know what to do about this as TS complains about it... Probably does not matter though, it may not affect UI at all.

Reply:
Well it is quite neat feature which makes the search persistent. i.e. when user search a word on the main page, the text stays in the search bar. It was one of the required features as I recall from our meetings. What is more fascinating is the fact that if I return the prop back, the whole site stops working, like completely, nothing is clickable. I have no idea how can this happen when in the previous version the prop is present and working.